src/digisos/informasjon/BehandlingAvPersonopplysningerModal.tsx

```diff
@@ -8,9 +8,14 @@ import {Soknadsdata} from "../redux/soknadsdata/soknadsdataReducer";
 import {State} from "../redux/reducers";
 import {visSamtykkeInfo} from "../redux/soknad/soknadActions";
 
+export const replaceNavkontor = (text: string, valgtEnhetsNavn?: string) => {
+    // Hvis ikke valgtEnhetsNavn finnes, erstattes søkestrengen med capture-gruppa ([\w\s-]*)
+    // som er 1 eller fler bokstaver, whitespace eller '-', feks "NAV-kontoret ditt" fra søknads-api-tekstene
+    return text.replaceAll(/{navkontor:([\w\s-]*)}/g, valgtEnhetsNavn ? valgtEnhetsNavn : "$1");
```

Comment:
.replaceAll er tydeligvis ikke en funksjon 🤔 
Fungerer denne i alle nettlesere?

Reply:
Byttet til replace! Trodde først det ikke funket, men nå ser det ut som det funket likevel. Skal test i labs for sikkerhetsskyld, må bare få bygget igjennom først :) 

(ble litt tidlig PR, hadde tenkt å lage draft, men fikk det ikke til i ettertid..)